### DISCUSSION TITLE
Idee schermontwerp

### DISCUSSION THREAD

**Discussion by jorrickstempher** [Windesheim-HBO-ICT/Curriculum#7] [Ideas]
- Schermontwerp laptop/pc
- Schermontwerp mobiel

_Het mobiele schermontwerp moet uiteindelijk gaan lijken op het uitklappen van folders in bv visual studio code._ 
Daarnaast moet er nog gekeken worden hoe de Wiki applicatie verwezen gaat worden in deze schermen. Eventuele voorbeelden/suggesties zijn van harte welkom :)

![crcl_schermontwerp_klein](https://github.com/Windesheim-HBO-ICT/Curriculum/assets/63049068/f9ad486a-2551-4990-8244-c353ca27fa60)
 
![crcl_schermontwerp_groot](https://github.com/Windesheim-HBO-ICT/Curriculum/assets/63049068/86f38b4f-c444-48d8-8848-f3c6e4212887)

**Comment by ernstbolt** [comment]
Helder uitgewerkt! Ik denk dat het schermontwerp voor mobiel afwijkt van het oorspronkelijke idee: ingesprongen lijsten te gebruiken. Kun je laten weten wat de overweging is geweest om van dit idee af te wijken?

**Reply by jorrickstempher** [reply]
Thanks! Dit was in eerste instantie hoe ik het in gedachten had, later bleek idd dat het niet helemaal overeenkomt met het oorspronkelijke idee. Moet ik dus a.s.a.p. een nieuw ontwerp voor maken :)

**Comment by jorrickstempher** [comment]
Nieuw mobiel schermontwerp gemaakt met (ik denk?) ingesprongen lijsten. Hier in ieder geval al een beter gevoel bij dan bovenstaande mobiele ontwerp :)

> Aan de rechterkant van het scherm staat een gele balk, deze kopjes hebben dan een link naar de wiki.
> Wellicht leuk/makkelijk om voor de SSDLC balk aan de linkerkant icoontjes te maken? Of dit ook verwerken in lijsten structuur?

![crcl_schermontwerp_klein_nieuw](https://github.com/Windesheim-HBO-ICT/Curriculum/assets/63049068/75f8207d-b605-41b5-aeb7-b448aafe28dd)

**Comment by jorrickstempher** [comment]
![ppdesign](https://github.com/Windesheim-HBO-ICT/Curriculum/assets/63049068/cb3eb0a5-0a28-4ef7-85eb-016383346c46)
![ppdesign2](https://github.com/Windesheim-HBO-ICT/Curriculum/assets/63049068/b840c920-e817-464a-a655-0db65c2caade)
![ppdesign3](https://github.com/Windesheim-HBO-ICT/Curriculum/assets/63049068/0eee67d6-8111-42f7-81c9-69b4e72b105a)

Voordat ik ga beginnen aan het verder uitwerken van de desktop versie van het Curriculum eerst even een betere 'wireframe' gemaakt. Dit is ongeveer wat ik in gedachten had, vergelijkbaar met de mobiele versie maar gewoon een grotere versie dus :) (ook de tags meegenomen hierin)

**Comment by ernstbolt** [comment]
Hoi Jorrick,

Ik neem aan dat je verder bent gegaan. Kun je de kleuren aanpassen naar het geel van Windesheim?

Met vriendelijke groeten,

Ernst

Van: Jorrick Stempher ***@***.***>
Verzonden: zondag 5 mei 2024 14:52
Aan: Windesheim-HBO-ICT/Curriculum ***@***.***>
CC: Ernst Bolt ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [Windesheim-HBO-ICT/Curriculum] Idee schermontwerp (Discussion #7)


ppdesign.png (view on web)<https://github.com/Windesheim-HBO-ICT/Curriculum/assets/63049068/cb3eb0a5-0a28-4ef7-85eb-016383346c46>
ppdesign2.png (view on web)<https://github.com/Windesheim-HBO-ICT/Curriculum/assets/63049068/b840c920-e817-464a-a655-0db65c2caade>
ppdesign3.png (view on web)<https://github.com/Windesheim-HBO-ICT/Curriculum/assets/63049068/0eee67d6-8111-42f7-81c9-69b4e72b105a>

Voordat ik ga beginnen aan het verder uitwerken van de desktop versie van het Curriculum eerst even een betere 'wireframe' gemaakt. Dit is ongeveer wat ik in gedachten had, vergelijkbaar met de mobiele versie maar gewoon een grotere versie dus :) (ook de tags meegenomen hierin)

-
